src/main/java/com/minecolonies/coremod/commands/AbstractSingleCommand.java

```diff
@@ -113,7 +113,7 @@ public static boolean isPlayerOpped(@NotNull final ICommandSender sender)
      * @param theCommand which command to check if the player can use it
      * @return boolean
      */
-    public boolean canCommandSenderUseCommand(final Commands theCommand)
+    public static boolean canCommandSenderUseCommand(final Commands theCommand)
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Reduce the number of returns of this method 14, down to the maximum allowed 6. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1142)